blueocean-core-js/src/js/components/ReplayButton.jsx

```diff
@@ -83,7 +83,7 @@ export class ReplayButton extends Component {
         return (
             <div className={`replay-button-component ${outerClass}`} onClick={(event => stopProp(event))}>
                 <a className={`replay-button ${innerButtonClass}`} title={replayLabel} onClick={() => this._onReplayClick()}>
-                    <Icon size={24} icon="replay" />
+                    <Icon size={24} icon="AvReplay" />
```

Comment:
The names are declared like this in "material-ui/svg-icons/index.js", I didn't want to change them because I wanted the update process to be as easy as possible in the future, I also created a story in the JDL storybook called Icon->all icons that renders all the icons with their respective names